### PR TITLE
[SSL] Updates to DCV info

### PR DESCRIPTION
**Pull request by kodster28**
For SSL for SaaS + regular CNAME setups. Addresses PCX-2738.

### CODE REVIEW

File: products/ssl/src/content/ssl-for-saas/common-tasks/certificate-validation-methods.md [hunk 1]
```diff
@@ -1,272 +1,115 @@
 ---
 order: 3
```
Comment (kodster28) [inline products/ssl/src/content/ssl-for-saas/common-tasks/certificate-validation-methods.md:2]: Make sure to review this file too! (didn't show up for me initially)

File: products/ssl/src/content/_partials/_email-validation-process.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+The addresses listed in this field will receive an email from `support@certvalidate.cloudflare.com`. They should either click **Review Certificate Request** or the https://certvalidate.cloudflare.com hyperlink.
+
+![Certificate Validation Email](../static/certvalidate-email.png)
+
+As soon as the domain owner has clicked the link in this email and clicked **Approve** on the validation page, the certificate will automatically transition from Pending Validation to Pending Issuance to Pending Deployment and finally Active.
```
Comment (dcpena) [inline products/ssl/src/content/_partials/_email-validation-process.md]: Should the different statuses be bolded?

File: products/ssl/src/content/_partials/_generic-validation-process.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Once you [create an advanced certificate](/edge-certificates/advanced-certificate-manager#create-a-certificate-in-the-dashboard) or [edit the `validation_method`](https://api.cloudflare.com/#ssl-verification-edit-ssl-certificate-pack-validation-method) via the API and use this validation method, you will see the following values (after waiting a few seconds):
```
Comment (dcpena) [inline products/ssl/src/content/_partials/_generic-validation-process.md]: Could possibly eliminate the parentheses and say "you will see the following values after a few seconds."

File: products/ssl/src/content/_partials/_ssl-for-saas-create-hostname.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Once you create a new hostname [via the dashboard](/ssl-for-saas/common-tasks/issuing-certificates#via-the-dashboard) or [via the API](/ssl-for-saas/common-tasks/issuing-certificates#via-the-api) and choose this validation method, you will see the following values (after waiting a few seconds):
```
Comment (dcpena) [inline products/ssl/src/content/_partials/_ssl-for-saas-create-hostname.md]: Comment from above applies here as well.

File: products/ssl/src/content/edge-certificates/advanced-certificate-manager.md [hunk 5]
```diff
@@ -134,6 +134,6 @@ For more information about DCV, refer to [DCV methods](/edge-certificates/univer
 
 <Aside type="warning">
 
-Due to recent changes, HTTP DCV validation will soon not be allowed for wildcard certificates. For more details and next steps, refer to [Changes to HTTP DCV](/reference/dcv-update).
+Due to recent changes, HTTP DCV validation will soon not be allowed for wildcard certificates or certificates with multiple SANs. For more details and next steps, refer to [Changes to HTTP DCV](/ssl-tls/dcv-update).
```
Comment (dcpena) [inline products/ssl/src/content/edge-certificates/advanced-certificate-manager.md]: Does the SAN acronym need spelled out here, or has that already been established?

File: products/ssl/src/content/edge-certificates/changing-dcv-method.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+<GenericValidationProcess/>
+
+- API: `emails`
+- Dashboard: When viewing an individual certificate at **SSL/TLS** > **Edge Certificates**, refer to the value for **Certificate validation email recipients**
+
+<EmailValidationProcess/>
+
+### Verify DCV status
+
+To verify the DCV status of a domain, either view the certificate in the dashboard or use the [Verification Status endpoing](https://api.cloudflare.com/#ssl-verification-ssl-verification-details).
```
Comment (dcpena) [inline products/ssl/src/content/edge-certificates/changing-dcv-method.md]:
```suggestion
To verify the DCV status of a domain, either view the certificate in the dashboard or use the [Verification Status endpoint](https://api.cloudflare.com/#ssl-verification-ssl-verification-details).
```

File: products/ssl/src/content/ssl-for-saas/common-tasks/certificate-validation-methods.md [hunk 7]
```diff
@@ -1,272 +1,115 @@
...
 
-<Aside type='note' header='Note'>
+<CreateHostname/>
 
-Cloudflare is able to serve the random token shown above from our edge due to the fact that site.example.com has a CNAME in place to `$CNAME_TARGET`, which ultimately resolves to Cloudflare IPs. If your customer has not yet added the CNAME, the attempt by the CA to retrieve that token will fail and the process will not complete.
+- [API](https://api.cloudflare.com/#custom-hostname-for-a-zone-custom-hostname-details): `txt_name` and `txt_value`
+- Dashboard: When viewing an individual certificate at **SSL/TLS** > **Custom Hostnames**, refer to the values for **Certificate validation TXT name** and **Certificate validation TXT value**
 
-We will attempt to retry this validation check for a finite period before timing out; see the <a href="/ssl-for-saas/validation-backoff-schedule">Validation Retry Schedule</a> for more details.
+Ask your customer to create a TXT record named the **name** and containing the **value** at their authoritative DNS provider. Once this TXT is in place, validation and certificate issuance will automatically complete.
```
Comment (dcpena) [inline products/ssl/src/content/ssl-for-saas/common-tasks/certificate-validation-methods.md]:
```suggestion
Ask your customer to create a TXT record named the **name** and containing the **value** at their authoritative DNS provider. Once this TXT record is in place, validation and certificate issuance will automatically complete.
```

File: products/ssl/src/content/ssl-for-saas/common-tasks/certificate-validation-methods.md [hunk 8]
```diff
@@ -1,272 +1,115 @@
...
 
-For example, if you create a new custom hostname for `site.example.com`, the CA might ask us to return the value `ca3-38734555d85e4421beb4a3e6d1645fe6` for a request to `http://site.example.com/.well-known/pki-validation/ca3-39f423f095be4983922ca0365308612d.txt"`. As soon as we receive that value from the CA we make it accessible at our edge and ask the CA to confirm it’s there so that they can complete validation and the certificate order.
+<TXTValidationDefinition/>
 
-<Aside type='note' header='Note'>
+<CreateHostname/>
 
-Cloudflare is able to serve the random token shown above from our edge due to the fact that site.example.com has a CNAME in place to `$CNAME_TARGET`, which ultimately resolves to Cloudflare IPs. If your customer has not yet added the CNAME, the attempt by the CA to retrieve that token will fail and the process will not complete.
+- [API](https://api.cloudflare.com/#custom-hostname-for-a-zone-custom-hostname-details): `txt_name` and `txt_value`
+- Dashboard: When viewing an individual certificate at **SSL/TLS** > **Custom Hostnames**, refer to the values for **Certificate validation TXT name** and **Certificate validation TXT value**
```
Comment (dcpena) [inline products/ssl/src/content/ssl-for-saas/common-tasks/certificate-validation-methods.md:33]:
```suggestion
- Dashboard: When viewing an individual certificate at **SSL/TLS** > **Custom Hostnames**, refer to the values for **Certificate validation TXT name** and **Certificate validation TXT value**.
```

File: products/ssl/src/content/ssl-for-saas/common-tasks/certificate-validation-methods.md [hunk 9]
```diff
@@ -1,272 +1,115 @@
...
 
---------
+<EmailValidationDefinition/>
 
-## Other
+<CreateHostname/>
 
-For your customers that already have HTTPS on their custom domain, e.g., they’re self-hosted or you’ve manually provisioned, and cannot tolerate a couple minutes of downtime during cutover, additional “pre-validation” methods are available.
+- [API](https://api.cloudflare.com/#custom-hostname-for-a-zone-custom-hostname-details): `emails`
+- Dashboard: When viewing an individual certificate at **SSL/TLS** > **Custom Hostnames**, refer to the value for **Certificate validation email recipients**
```
Comment (dcpena) [inline products/ssl/src/content/ssl-for-saas/common-tasks/certificate-validation-methods.md]:
```suggestion
- Dashboard: When viewing an individual certificate at **SSL/TLS** > **Custom Hostnames**, refer to the value for **Certificate validation email recipients**.
```

File: products/ssl/src/content/ssl-for-saas/common-tasks/certificate-validation-methods.md [hunk 10]
```diff
@@ -1,272 +1,115 @@
...
-$ curl -X PATCH "https://api.cloudflare.com/client/v4/zones/{zone_id}/custom_hostnames/46f8849a-72c9-49e0-9e42-857297d89306" \
-       -H "X-Auth-Email: {email}" -H "X-Auth-Key: {key}" \
-       -H "Content-Type: application/json" \
-       -d '{"hostname":"another.example.com", "ssl":{"method":"txt","type":"dv"}}'
-```
+<CreateHostname/>
 
-### Email
+- [API](https://api.cloudflare.com/#custom-hostname-for-a-zone-custom-hostname-details): `http_url` and `http_body`
+- Dashboard: When viewing an individual certificate at **SSL/TLS** > **Custom Hostnames**, refer to the values for **Certificate validation request** and **Certificate validation response**
```
Comment (dcpena) [inline products/ssl/src/content/ssl-for-saas/common-tasks/certificate-validation-methods.md]:
```suggestion
- Dashboard: When viewing an individual certificate at **SSL/TLS** > **Custom Hostnames**, refer to the values for **Certificate validation request** and **Certificate validation response**.
```

**Review by dcpena — APPROVED** [review]

File: products/ssl/src/content/edge-certificates/changing-dcv-method.md (hunk 6 above)
Comment (kodster28) [reply products/ssl/src/content/edge-certificates/changing-dcv-method.md]: Oops!

**Pull request merged**